runtime/stackmap/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+################################################################################
+# Copyright (c) 2017, 2017 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+################################################################################
+
+add_tracegen(map.tdf)
+
+add_library(j9stackmap STATIC
+	debuglocalmap.c
+	fixreturns.c
+	localmap.c
+	mapmemorybuffer.c
+	maxmap.c
+	stackmap.c
+	${CMAKE_CURRENT_BINARY_DIR}/ut_map.c
```

Comment:
Is this a 3rd(?) way to deal with tracefiles?

Reply:
No  this is just the first way, but more explicit.  I think the first way falls apart when the tracegen'd files dont actually live in the source tree. However it still works now because we are still running tracegen as before (thus making add_tracgen superfluous )